src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -128,14 +125,16 @@ public class CallbackController {
     private final DocumentGenService documentGenService;
     private final SendgridService sendgridService;
     private final C100JsonMapper c100JsonMapper;
-    private final AuthTokenGenerator authTokenGenerator;
     private final CourtFinderService courtLocatorService;
     private final LocationRefDataService locationRefDataService;
     private final UpdatePartyDetailsService updatePartyDetailsService;
     private final PaymentRequestService paymentRequestService;
     private final CourtSealFinderService courtSealFinderService;
     private final ConfidentialityTabService confidentialityTabService;
     private final LaunchDarklyClient launchDarklyClient;
+
+    @Qualifier("caseSummaryTab")
```

Comment:
why do we need a qualifier?

Reply:
We have @Qualifier present in CaseSummaryTabService, hence add the same here